client/my-sites/posts/main.jsx

```diff
@@ -73,12 +73,11 @@ const PostsMain = React.createClass( {
 					siteId={ site.ID }
 					query={ this.props.draftsQuery } />
 				<QueryPostCounts siteId={ site.ID } type="post" />
-				<Card compact className="posts__drafts-header">
-					{ this.translate( 'Latest Drafts' ) }
-					<Button borderless href={ this.props.newPostPath }>
-						<Gridicon icon="plus" />
+				<SectionHeader className="posts__drafts-header" label={ this.translate( 'Latest Drafts' ) }>
+					<Button compact href={ this.props.newPostPath }>
+						{ this.translate( 'Start New' ) }
 					</Button>
-				</Card>
+				</SectionHeader>
 				{ this.props.drafts && this.props.drafts.map( this.renderDraft, this ) }
```

Comment:
Wonder if we should move `renderDraft` inline? `_.map` can guard against `null` `drafts`, and I don't think the `! draft` is catching anything. Proposed:

```js
{ map( this.props.drafts, ( draft ) => (
	<PostItem
		compact
		key={ draft.global_ID }
		globalId={ draft.global_ID } />
) ) }
```

Or:

```js
{ map( this.props.drafts, ( { global_ID: globalId } ) => (
	<PostItem compact key={ globalId } globalId={ globalId } />
) ) }
```